operator/cmd/mesh/operator-init.go

```diff
@@ -51,7 +51,7 @@ func addOperatorInitFlags(cmd *cobra.Command, args *operatorInitArgs) {
 	cmd.PersistentFlags().StringVar(&args.common.operatorNamespace, "operatorNamespace", operatorDefaultNamespace,
 		"The namespace the operator controller is installed into")
 	cmd.PersistentFlags().StringVar(&args.common.istioNamespace, "istioNamespace", istioDefaultNamespace,
```

Comment:
I think we should rename the CLI argument to `watchedNamespaces` or similar, the old name will be misleading

Reply:
agreed, renamed to `watchedNamespaces`

Reply:
thanks very much